interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a gem?
 
-Your answer:
+Your answer: A gem in ruby is small portion of code that will be used often that creates functionality to avoid the developer having to retype code over and over.
 
-Researched answer:
+Researched answer: A gem is a snippet of code functionality. They will implement commonly used functions to an application. They also make it easier to share functionality. 
 
 2. What does the acronym CRUD stand for?
 
-Your answer:
+Your answer: CRUD stands for Create, Read, Update, and Delete. It refers to operations that are available to use in Rails
 
-Researched answer:
+Researched answer: Create, Read, Update, Delete. It refers to the four functions that are considered necessary to implement an application.
 
 3. What is Ruby on Rails?
 
-Your answer:
+Your answer: Ruby on rails is a development program that used the Ruby language of coding and puts it together with Rails methods.
 
-Researched answer:
+Researched answer: It is an open-source web application development framework written in the Ruby programming language. It provides a pre-built structure for development and everything you need to build a web app.
 
 4. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is something that you can input data, and store it in different categories and you can update individual items in the database without having to access the entire code.
 
-Researched answer:
+Researched answer: It is a type of database that stores and provides access to data points that are related to one another. In a relational database, each row in the table is a record with a unique ID called the key. The columns of the table hold attributes of the data, and each record usually has a value for each attribute, making it easy to establish the relationships among data points.
 
 5. What are primary keys? Why are they important?
 
-Your answer:
+Your answer: The primary key is the way that data withing a database is categorized. For example a primary key for "name" would be where you would store all of the names in the database.
 
-Researched answer:
+Researched answer: Each row is has a primary key named "id" these are always unique. This is important so you can access the correct data even if the information in that row is similar to information in another row.
```

Comment:
Nice work in this section.  Just a reminder, when possible try to extend the answer by providing examples.